build/gen_stub.php

```diff
@@ -988,6 +1352,10 @@ class FileInfo {
     public $declarationPrefix = "";
     /** @var bool */
     public $generateLegacyArginfo = false;
+    /** @var bool */
+    public $generateClassEntries = false;
+    /** @var bool */
+    public $generateLegacyClassEntries = false;
```

Comment:
Appear to be unused, and I think we'd want legacy-arginfo to cover this as well.

Reply:
As it turned out, I've already added support for it, only a few conditions containing `$generateLegacyClassEntries` were missing :)

Reply:
I guess my main concern here was why we need two different options. If you either want to generate code compatible with older versions or not, doesn't make much sense to generate arginfo for PHP 7 but classes for PHP 8, right? Or do I misunderstand the purpose?

Reply:
Oh, now I got it! Yeah, you were right, the current code will work well without `$generateLegacyClassEntries`.